src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private double side;
+
+    public Square(double side, String color) {
+        this.setColor(color);
+        this.side = side;
+    }
+
+    @Override
+    public double getArea() {
+        double area;
+        area = this.side * this.side;
+        return area;
```

Comment:
```suggestion
        return this.side * this.side;
```
dont create redundant variables